docs/user-management/rbac/projects.md

```diff
@@ -0,0 +1,58 @@
+---
+pageType: howto
+title: RBAC projects
+description: Understand how n8n uses project for RBAC. Learn how to create and manage projects.
+---
+
+/// info | Feature availability
+RBAC is available on all plans except the Community edition. Different plans have different numbers of projects and roles:
+
+* Starter: one project, with admin role.
+* Pro: three projects, with admin and editor roles.
+* Enterprise: unlimited projects, with admin and editor roles.
```

Comment:
Same information can be found in `docs/user-management/rbac/index.md` so maybe to avoid duplication this could be extracted - also one thing I wonder is whether we should omit this from here and just direct people to the pricing page?

Reply:
Good plan